src/app/backend-api.service.ts

```diff
@@ -11,7 +11,7 @@ import {GlobalVarsService} from './global-vars.service';
   providedIn: 'root'
 })
 export class BackendAPIService {
-  endpoint = `https://${environment.nodeHostname}/api/v0`;
+  endpoint = `http://${environment.nodeHostname}/api/v0`;
```

Comment:
I looked at the backend API endpoints and this seems fine but given that identity is very security sensitive, it seems undesirable to downgrade this to http.  Is this because localhost and https don't play nicely together?  If so, maybe we restrict this to only be http on localhost?